src/ostree/ot-builtin-commit.c

```diff
@@ -78,6 +81,9 @@ static GOptionEntry options[] = {
   { "body", 'm', 0, G_OPTION_ARG_STRING, &opt_body, "Full description", "BODY" },
   { "body-file", 'F', 0, G_OPTION_ARG_FILENAME, &opt_body_file, "Commit message from FILE path", "FILE" },
   { "editor", 'e', 0, G_OPTION_ARG_NONE, &opt_editor, "Use an editor to write the commit message", NULL },
+#ifdef OSTREE_ENABLE_EXPERIMENTAL_API
+  { "collection-id", 'c', 0, G_OPTION_ARG_STRING, &opt_collection_id, "Collection ID", "ID" },
```

Comment:
I think the other commands have set the placeholder (last element) to `COLLECTION-ID` rather than `ID`.